ownCloud/UI Elements/RefreshControl.swift

```diff
@@ -0,0 +1,56 @@
+//
```

Comment:
@pablocarmu if possible we should not introduce hacks like this and subclass UI classes which are not supposed to be subclassed

Reply:
@pablocarmu Did you try to fix it by setting UITableView background color eventually?

Reply:
Yes, that was my first approach, but it only fixes the color problem. The "blink" of the table view problem still persisted until I made the _hack_ on `UIRefreshControl`